qchem/pennylane_qchem/qchem/hf/basis_data.py

```diff
@@ -138,3 +138,150 @@
         ],
     },
 }
+
+POPLE631G = {
```

Comment:
Just want to know where do we get these data from?

Reply:
From the Basis Set Exchange [library](https://www.basissetexchange.org), which is cited at the beginning of the file. It is pip [installable](https://pypi.org/project/basis-set-exchange/).